provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -275,11 +275,13 @@ def _create_node(self, machine_id: str) -> NodeDesc:
         # Adding a Motr confd entry per server node in CDF.
         # The `runs_confd` value (true/false) determines if Motr confd process
         # will be started on the node or not.
-        servers.value.append(M0ServerDesc(
-            io_disks=DisksDesc(
-                data=DList([], 'List Text'),
-                meta_data=Maybe(None, 'Text')),
-            runs_confd=Maybe(True, 'Bool')))
+        start_confd = store.get(f'server_node>{machine_id}>consul_server')
```

Comment:
Can we check like this,
if 'True' == store.get(f'server_node>{machine_id}>consul_server')

Reply:
What for?

Reply:
I don't think it makes sense. What is more important, 'True' is a raw string which can cause more problems as I described above. If compared to that problem, one-line economy doesn't look necessary.